Doc/library/http.cookies.rst

```diff
@@ -129,17 +129,17 @@ Morsel Objects
    Abstract a key/value pair, which has some :rfc:`2109` attributes.
 
    Morsels are dictionary-like objects, whose set of keys is constant --- the valid
-   :rfc:`2109` attributes, which are
-
-   * ``expires``
-   * ``path``
-   * ``comment``
-   * ``domain``
-   * ``max-age``
-   * ``secure``
-   * ``version``
-   * ``httponly``
-   * ``samesite``
+   :rfc:`2109` attributes, which are:
+
+   * .. attribute:: expires
+   * .. attribute:: path
+   * .. attribute:: comment
+   * .. attribute:: domain
+   * .. attribute:: max-age
+   * .. attribute:: secure
+   * .. attribute:: version
+   * .. attribute:: httponly
+   * .. attribute:: samesite
```

Comment:
Currently this is rendered as a nice compact list by Sphinx:

<details>
<summary>Screenshot</summary>

![image](https://github.com/python/cpython/assets/66076021/94ffcb2d-a12e-46cf-b0bc-23dfe42899f5)

</details>

Unfortunately with this PR, Sphinx adds double spacing between the bullets :(

<details>
<summary>Screenshot</summary>

![image](https://github.com/python/cpython/assets/66076021/d3cdd4f6-1335-4d47-9d37-014430807ed8)

</details>

If we did this instead, then we still get the link anchors, but we also get nice compact formatting like we currently have on `main`. We lose the bullet points, but I can't figure out a way of keeping the compact formatting while also keeping the bullets, unfortunately:

```suggestion
     .. attribute:: expires
                    path
                    comment
                    domain
                    max-age
                    secure
                    version
                    httponly
                    samesite
```

How it renders:

<details>
<summary>Screenshot</summary>

![image](https://github.com/python/cpython/assets/66076021/1f5c5a6e-d464-4ecf-9669-6f3709a8f775)

</details>